applications/webrtc_video_server/webrtc_server.py

```
@@ -114,26 +165,60 @@ async def _javascript(self, request):
         loop = asyncio.new_event_loop()
         asyncio.set_event_loop(loop)
         loop.run_until_complete(self._runner.setup())
         site = web.TCPSite(self._runner, self._host, self._port, ssl_context=self._ssl_context)
-        logging.info(f"Starting web server at {self._host}:{self._port}")
+
+        cors_status = "with aiohttp-cors" if CORS_AVAILABLE else "with manual CORS headers"
+        mode = "with embedded client" if self._serve_client else "API-only mode"
+        logging.info(f"Starting WebRTC server at {self._host}:{self._port} ({mode}, {cors_status})")
+        if self._serve_client:
+            logging.info(f"  Web UI: http://{self._host}:{self._port}/")
+        logging.info(f"  API endpoints:")
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)